tests/resource/resource_log_samples/alter_resource_alter_audit_by_audit_alter_access_error.log

```diff
@@ -0,0 +1,70 @@
+                                                                               
```

Comment:
All of these debug log files should be able to be deleted now that we are using mock to verify the behavior of the audit rules setters using mocks and the generated security request dictionaries, no?

Reply:
yes, sorry. I forgot to take them out after refactoring.